src/VisualStudio/IntegrationTest/TestUtilities/InProcess/VisualStudioWorkspace_InProc.cs

```diff
@@ -81,5 +83,23 @@ public void CleanUpWaitingService()
 
                 GetWaitingService().EnableActiveTokenTracking(true);
             });
+
+        public void SetFeatureOption(string feature, string optionName, string language, string valueString)
```

Comment:
Not sure, but probably? If so, this should be in a try/finally. @tannergooding might know off the top of his head.